_includes/try-it.md

```diff
@@ -1,3 +1,286 @@
+Metal³ Development Environment
+==============================
+
+Before you begin
+------------
+
+Metal³-dev-env is aimed to set up an
+emulated environment which creates a set of Virtual Machines (VMs) to manage as
+if they were bare metal hosts.
+
+* [Before you begin](#before-you-begin)
+   * [Architecture](#architecture)
+   * [Prerequisites](#prerequisites)
+   * [Environment variables](#environment-variables)
+* [Quick start](#quick-start)
+   * [Deploying Metal3-dev-env](#deploy-metal3-dev-env)
+   * [Provision Cluster and Machines](#provision-cluster-and-machines)
+   * [Deprovision Cluster and Machines](#deprovision-cluster-and-machines)
+   * [Tear down Metal3-dev-env](#tear-down-metal3-dev-env)
+
```

Comment:
What about adding the detailed step by step instructions there too ? To remove the duplicated TOC